pkgs/development/python-modules/objax/default.nix

```diff
@@ -40,6 +51,23 @@ buildPythonPackage rec {
     "objax"
   ];
 
+  catchConflicts = false;
```

Comment:
let's add a code comment explaining this setting, since it's a little unconventional

Reply:
Is that a problem to leave that in the main scope ? Or should we declare a `passthru.tests` attrs ?

Reply:
I added a comment. Feel free to suggest another phrasing.

Reply:
LGTM!